### ISSUE TITLE
hot-update.js is included in output JSON with webpack 5

### ISSUE THREAD

**Issue by SPWizard01** [ztoben/assets-webpack-plugin#273]
**Describe the bug**
assets webpack plugin include hot update modules
i.e. when using HMR first json file result:
```
{"someEntry":{"js":["js/runtime.js?7eefe4319e6e5d92569e","js/app.js?95ae18e01b0b1cf8607a"]}}
```
after updating any file consequent results of json file:
```
{"someEntry":{"js":["js/runtime.js?7eefe4319e6e5d92569e","runtime.4c3e8270b8cf8c430681.hot-update.js","js/app.js?95ae18e01b0b1cf8607a"]}}
```

which results in the following in the browser:
```
jsonp chunk loading:103 Uncaught TypeError: Cannot read property 'push' of undefined
    at self.webpackHotUpdatePACKAGEJSONNAME (jsonp chunk loading:103)
```

**To Reproduce**
run webpack-dev-server with `hot: true`

**Expected behavior**
i am not sure if hot-update.js should be included.

**Webpack Config**
simple config will suffice
```
    new AssetsPlugin({
      path: resolve(__dirname, "./dist"),
      removeFullPathAutoPrefix: true,
      fullPath: false,
      entrypoints: true,
      filename: "webpack.assets",
      keepInMemory: false,
    } as any),
```
**Desktop (please complete the following information):**
 - OS: Windows 10
 - Node version: v12.18.0
 - Plugin version: 6.0.0-alpha.3

**Additional context**
webpack@latest
webpack-cli@latest
webpack-dev-server@latest
i am not sure if hot-update.js should be included, seems like not.

**Comment by ztoben** [comment]
Looks like the name of the update is served with `[fullhash]` now rather than `[hash]`. Should be a pretty easy fix.

**Comment by ztoben** [comment]
@SPWizard01 I think this should be solved with `6.0.0-alpha.5`. Let me know if it's still an issue.
